### PR TITLE
[TaxationBundle] DelegatingCalculator use the ServiceRegistry

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

@Sylius/core-team what do you think ?

### CODE REVIEW

File: src/Sylius/Bundle/TaxationBundle/Resources/config/services.xml [hunk 1]
```diff
@@ -19,30 +19,22 @@
...
-            <tag name="form.type" alias="sylius_tax_rate" />
-        </service>
-
         <service id="sylius.form.type.tax_calculator_choice" class="%sylius.form.type.tax_calculator_choice.class%">
             <argument>%sylius.tax_calculators%</argument>
             <tag name="form.type" alias="sylius_tax_calculator_choice" />
         </service>
 
-        <service id="sylius.tax_calculator" class="%sylius.tax_calculator.class%" />
+        <service id="sylius.tax_calculator" class="%sylius.tax_calculator.class%">
```
Comment (aramalipoor) [inline src/Sylius/Bundle/TaxationBundle/Resources/config/services.xml]: Isn't it better to use `sylius.registry.tax_calculator` ?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/TaxationBundle/Resources/config/services.xml]: :+1:

**Comment by aRn0D** [comment]
@pjedrzejewski @aramalipoor updated !

**Comment by aRn0D** [comment]
@pjedrzejewski I will revert hte deletion of the `DelegatingCalculator` because I want to use it like the `ShippingBundle`

**Comment by aRn0D** [comment]
@Sylius/core-team / @michalmarcinkowski any feedback ?

**Comment by aRn0D** [comment]
@pjedrzejewski any feedback ?

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by pjedrzejewski** [comment]
How many pings it takes to get something merged? Answer: Ping! :dancer: Thank you Arnaud!

**Pull request merged**